components/Header.js

```diff
@@ -47,47 +90,42 @@ class Header extends Component {
           >
             Menu
           </button>
-          <nav
-            className={this.state.menuHidden ? 'hidden main-menu-wrapper' : 'visible main-menu-wrapper'}
-            onClick={this.handleMenuToggle}
-          >
+          <nav className={menuHidden ? 'hidden main-menu-wrapper' : 'visible main-menu-wrapper'}>
             <ul>
-              <li className="has-submenu">
-                <button type="button" className="btn--link">
-                  About
-                </button>
-                <ul className="submenu">
-                  <li>
-                    <Link href="/about">
-                      <a>About Us</a>
-                    </Link>
-                  </li>
-                  <li>
-                    <Link href="/about-the-data">
-                      <a>About the Data</a>
-                    </Link>
-                  </li>
-                  <li>
-                    <Link href="/related-organizations">
-                      <a>Related Organizations</a>
-                    </Link>
-                  </li>
-                </ul>
+              <li className="desktop-about">
+                <details ref={this.aboutMenu}>
+                  <summary className="btn--link">About</summary>
+                  <ul>
+                    <AboutLinks onLinkClick={this.handleMenuLinkClick} onLinkKeyDown={this.handleMenuLinkKeyDown} />
+                  </ul>
+                </details>
               </li>
+              <div className="mobile-about">
+                <AboutLinks onLinkClick={this.handleMenuLinkClick} onLinkKeyDown={this.handleMenuLinkKeyDown} />
```

Comment:
There's a little bit of duplication here for mobile and desktop, but trying to style and properly open/close the `<details>` element on mobile views was a huge pain, so instead, we just won't show it on mobile.